src/components/AuthorisationForm/Login.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import s from './Authorisation.module.css';
+import Input from './FormElements/Input';
+import SubmitButton from './FormElements/SubmitButton';
+import { basicSchema } from '../schemas/schema';
+import SuccessAlert from './FormElements/SuccessAlert';
+
+const onSubmit = async (values, actions) => {
+    // eslint-disable-next-line no-promise-executor-return
+    await new Promise(resolve => setTimeout(resolve, 4500));
+    actions.resetForm();
+};
+export default function Login() {
+    const {
+        values,
+        isSubmitting, touched,
+        errors, handleBlur,
+        handleChange,
+        handleSubmit } = useFormik({
+        initialValues: {
+            email: '',
+            age: '',
+            password: '',
+            confirmPassword: '',
+        },
+        validationSchema: basicSchema,
+        onSubmit,
+    });
+
+    return (
+        <form onSubmit={handleSubmit} className={s.form} autoComplete="off">
+            {isSubmitting && <SuccessAlert />}
+            <Input
+                label="Email" value={values.email} change={handleChange}
+                blur={handleBlur} type="email" placeholder="Enter your email"
+                name="email" className={errors.email && touched.email && `${s.inputError}`}
+            />
+            {errors.email && touched.email && <p className={s.error}>{errors.email}</p>}
```

Comment:
Make an ErrorMessage component that will be unique so you won't duplicate the code